packages/python2-libvhdi-python/PKGBUILD

```diff
@@ -0,0 +1,28 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python2-libvhdi-python
+pkgver=20181227
+pkgrel=1
+pkgdesc='Library and tools to access the Virtual Hard Disk (VHD) image format.'
+groups=('blackarch' 'blackarch-misc')
+url='https://github.com/libyal/libvhdi'
+arch=('x86_64' 'armv6h' 'armv7h' 'aarch64')
+license=('LGPLv3+')
+depends=('python2')
+makedepends=('python2' 'python2-setuptools' 'gcc')
```

Comment:
`python2` shouldn't be here since it's already in `depends`.

Reply:
ACK